plugins/cstrike/miscstats.sma

```diff
@@ -543,48 +537,26 @@ public client_death(killer, victim, wpnindex, hitplace, TK)
 		}
 	}
 
-	new team = get_user_team(victim)
-	if (EnemyRemaining && is_user_connected(victim))
+	new team = get_user_team(victim);
```

Comment:
`cs_get_user_team`?

Reply:
It's actually weird this native was not used from the start.